lib/nerves_network/eapol_manager.ex

```diff
@@ -377,6 +394,15 @@ defmodule Nerves.Network.EAPoLManager do
     iex> is_map(retval)
     true
 
+    iex> setup = %{
```

Comment:
This should be an in-file UT, not a doctest.

Reply:
I disagree, the provided example illustrates the use of the API and that's the goal